<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSettingsEditor</class>
 <widget class="QMainWindow" name="ProjectSettingsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>9</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>544</width>
         <height>994</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_UserConfig">
          <property name="title">
           <string>User config</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="checkBox_UsePoryscript">
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Use Poryscript</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_ShowWildEncounterTables">
             <property name="text">
              <string>Show Wild Encounter Tables</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_DefaultTilesets">
          <property name="title">
           <string>Default Tilesets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_NewMapDefaults">
          <property name="title">
           <string>New Map Defaults</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_BorderMetatiles"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_Elevation">
             <property name="toolTip">
              <string>The default elevation that will be used to fill new maps</string>
             </property>
             <property name="text">
              <string>Elevation</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QCheckBox" name="checkBox_CreateTextFile">
             <property name="toolTip">
              <string>Whether a separate text.inc or text.pory file will be created for new maps, alongside the scripts file</string>
             </property>
             <property name="text">
              <string>Create separate text file</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinBox_Elevation"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_FillMetatile">
             <property name="toolTip">
              <string>The default metatile value that will be used to fill new maps</string>
             </property>
             <property name="text">
              <string>Fill Metatile</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_BorderMetatiles">
             <property name="toolTip">
              <string>A comma-separated list of metatile values that will be used to fill new map borders</string>
             </property>
             <property name="text">
              <string>Border Metatiles</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_FillMetatile">
             <property name="prefix">
              <string>0x</string>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_Prefabs">
          <property name="title">
           <string>Prefabs</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_PrefabsPath">
             <property name="text">
              <string>Prefabs Path</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_PrefabImportPrompted">
             <property name="text">
              <string>Prefab import prompted</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_PrefabsPath"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_Warning">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Warning">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:13pt; color:#d7000c;&quot;&gt;WARNING:  &lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;The settings from this point below require project changes to function properly. Do not modify these settings without the necessary changes. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_BaseGameVersionBox" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QWidget" name="widget_BaseGameVersion" native="true">
             <layout class="QFormLayout" name="layout_BaseGameVersion"/>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_TilesetsMetatiles">
          <property name="title">
           <string>Tilesets / Metatiles</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QLabel" name="label_AttributesSize">
             <property name="toolTip">
              <string>The number of bytes used per metatile for metatile attributes</string>
             </property>
             <property name="text">
              <string>Attributes size (in bytes)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_TerrainTypeMask">
             <property name="toolTip">
              <string>The mask used to read/write Terrain Type from the metatile's attributes data. If 0, this attribute is disabled.</string>
             </property>
             <property name="text">
              <string>Terrain Type mask</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_EnableTripleLayerMetatiles">
             <property name="text">
              <string>Enable Triple Layer Metatiles</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QWidget" name="widget_SizeDropdown" native="true"/>
           </item>
           <item row="10" column="0">
            <widget class="QCheckBox" name="checkBox_OutputIsCompressed">
             <property name="toolTip">
              <string>Whether the C data outputted for new tilesets will include the &quot;isCompressed&quot; field</string>
             </property>
             <property name="text">
              <string>Output 'isCompressed' field</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="spinBox_EncounterTypeMask">
             <property name="prefix">
              <string>0x</string>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="spinBox_LayerTypeMask">
             <property name="prefix">
              <string>0x</string>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_EncounterTypeMask">
             <property name="toolTip">
              <string>The mask used to read/write Encounter Type from the metatile's attributes data. If 0, this attribute is disabled.</string>
             </property>
             <property name="text">
              <string>Encounter Type mask</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QCheckBox" name="checkBox_OutputCallback">
             <property name="toolTip">
              <string>Whether the C data outputted for new tilesets will include the &quot;callback&quot; field</string>
             </property>
             <property name="text">
              <string>Output 'callback' field</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_BehaviorMask">
             <property name="toolTip">
              <string>The mask used to read/write Metatile Behavior from the metatile's attributes data. If 0, this attribute is disabled.</string>
             </property>
             <property name="text">
              <string>Behavior mask</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_LayerTypeMask">
             <property name="toolTip">
              <string>The mask used to read/write Layer Type from the metatile's attributes data. If 0, this attribute is disabled.</string>
             </property>
             <property name="text">
              <string>Layer Type mask</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="spinBox_BehaviorMask">
             <property name="prefix">
              <string>0x</string>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="spinBox_TerrainTypeMask">
             <property name="prefix">
              <string>0x</string>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_Events">
          <property name="title">
           <string>Events</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBox_EnableRequiresItemfinder">
             <property name="text">
              <string>Enable 'Requires Itemfinder' for Hidden Items</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox_EnableQuantity">
             <property name="text">
              <string>Enable 'Quantity' for Hidden Items</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_EnableSecretBases">
             <property name="text">
              <string>Enable Secret Bases</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_EnableWeatherTriggers">
             <property name="text">
              <string>Enable Weather Triggers</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_EnableCloneObjects">
             <property name="text">
              <string>Enable Clone Objects</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_EnableRespawn">
             <property name="text">
              <string>Enable 'Respawn Map/NPC' for Heal Locations</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_Maps">
          <property name="title">
           <string>Maps</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox_EnableAllowFlags">
             <property name="toolTip">
              <string>Whether &quot;Allow Running&quot;, &quot;Allow Biking&quot; and &quot;Allow Dig &amp; Escape Rope&quot; are default options for Map Headers</string>
             </property>
             <property name="text">
              <string>Enable 'Allow Running/Biking/Escaping'</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_EnableFloorNumber">
             <property name="toolTip">
              <string>Whether &quot;Floor Number&quot; is a default option for Map Headers</string>
             </property>
             <property name="text">
              <string>Enable 'Floor Number'</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_EnableCustomBorderSize">
             <property name="toolTip">
              <string>Whether the dimensions of the border can be changed. If not set, all borders are 2x2</string>
             </property>
             <property name="text">
              <string>Enable Custom Border Size</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
